### PR TITLE
[Bromley] waste phase2 staff only

### PR DESCRIPTION
**Pull request by dracos**
Adds garden_disabled feature to disable it entirely, or garden_waste_staff_only to restrict to staff. Only in terms of what is displayed to the user. [skip changelog]

### CODE REVIEW

File: templates/web/base/waste/bin_days.html [hunk 1]
```diff
@@ -96,7 +96,7 @@ <h3>More services</h3>
             [% IF any_request_allowed %]
               <li><a href="[% c.uri_for_action('waste/request', [ property.id ]) %]">Request a new container</a></li>
             [% END %]
-            [% IF !pending_subscription AND !services_available.Garden_Waste.is_active %]
+            [% IF NOT waste_features.garden_disabled AND NOT pending_subscription AND NOT services_available.Garden_Waste.is_active %]
```
Comment (struan) [inline templates/web/base/waste/bin_days.html:99]: garden_disabled is never set here if there isn't a garden subscription so this will still show.

**Review by struan — CHANGES_REQUESTED** [review]
one thing with the subscribe link

File: templates/web/base/waste/bin_days.html (hunk 1 above)
Comment (dracos) [reply templates/web/base/waste/bin_days.html:99]: Oh yes, good point

**Review by struan — APPROVED** [review]
👍

**Pull request merged**